whitepapers/0007_governor.md

```diff
@@ -4,37 +4,88 @@
 
-- Set a blanket rate limiting on all supported chains for all tokens
-- Prevent any single "bad actor" from blocking other value transfer by intentionally exceeding the transfer limit for the given time period
+If token bridge transfers are unlimited and instantaneous, a bug in a single connected chain could cause the entire token bridge to be drained.
 
-## Overview
+## Goals
 
-Each individual Guardian within the Guardian network should employ a set of strategies to verify the validity of a VAA. The Governor is designed to check VAAs that transfer tokens by enforcing limits on the notional value that can be transferred from a given chain over a specific period of time.
+If a Guardian decides to enable this feature:
+* Delay Wormhole messages from registered token bridges for 24h, if their notional value is excessively large.
+* This gives the Guardian the opportunity to delete pending messages, if they were created through a software bug and not accurately represent the state of the origin chain.
+* Protect against sybil attacks.
 
-The current implementation works on two classes of transaction (large and small) and current configuration can be found [here](https://github.com/wormhole-foundation/wormhole/blob/main/node/pkg/governor/mainnet_chains.go):
+## Non-Goals
 
-- **Large Transactions**
-  - A transaction is large if it is greater than or equal to the `bigTransactionSize` for a given origin chain.
-  - All large transactions will have a mandatory 24-hour finality delay and will have no affect on the `dailyLimit`.
-- **Small Transactions**
-  - A transaction is small if it is less than the `bigTransactionSize` for a given origin chain.
-  - All small transactions will have no additional finality delay up to the `dailyLimit` defined within a 24hr sliding window.
-  - If a small transaction exceeds the `dailyLimit`it will be delayed until it either
-    - fits inside the `dailyLimit` and will be counted toward the `dailyLimit`
-    - has been delayed for 24-hours and will have no affect on the `dailyLimit`.
+* Synchronize state between Guardians. Each Guardian may have a slightly different view of the network, including Governor configuration, ordering and completeness of token bridge transfers, etc. The Governor is a local feature that makes decisions based on each Guardian's individual view of the network.
+* Prevent quality-of-service degradation attacks where one bad actor causes a majority of transfers to be delayed.
+
+## High-Level Design
+
+### Delay Decision Logic
+*Configuration*: For each chain, a _single-transaction-threshold_ and a *24h-threshold* denominated in a base currency (e.g. USD) is specified [here](https://github.com/wormhole-foundation/wormhole/blob/main/node/pkg/governor/mainnet_chains.go).
```

Comment:
Can you explicitly state that the governor only monitors chains and tokens that are in the configuration?